src/helpers/getLocaleFromRequest/index.js

```diff
@@ -17,7 +17,11 @@ function getQueryFromRequest (request) {
       return param
     }
 
-    return request.query[key] ? decodeURIComponent(request.query[key]) : null
+    try {
+      return request.query[key] ? decodeURIComponent(request.query[key]) : null
+    } catch (error) {
+      return null
+    }
```

Comment:
I prefered when you let it fail before but doesn’t matter 😄

Reply:
Well but that’s the entire problem.

Reply:
No. What I meant is, before you rebased you wrote:

```javascript
try {
  return decodeURIComponent(request.query[key])
} catch (error) {
  return null
}
```

Reply:
Oh but that throw an error. Ok :)

Reply:
That doesn’t do the same thing, all tests were failing. `decodeURIComponent(undefined)
` === `'undefined'`.